python/pyhail/dataset.py

```diff
@@ -930,10 +1199,115 @@ def linreg(self, y, covariates='', root='va.linreg', minac=1, minaf=None):
             pargs.append(str(minaf))
         return self.hc.run_command(self, pargs)
 
-    def logreg(self, test, y, covariates=None, root=None):
+    def logreg(self, test, y, covariates=None, root='va.logreg'):
         """Test each variant for association using the logistic regression
         model.
 
+        **Example**
+
+        Run logistic regression with Wald test with two covariates:
+
+        >>> (hc.read('data/example.vds')
+        >>>   .annotate_samples_table('data/pheno.tsv', root='sa.pheno',
+        >>>     config=TextTableConfig(impute=True))
+        >>>   .logreg('wald', 'sa.pheno.isCase', covariates='sa.pheno.age, sa.pheno.isFemale'))
+
+        **Notes**
+
+        The :py:meth:`~pyhail.VariantDataset.logreg` command performs,
+        for each variant, a significance test of the genotype in
+        predicting a binary (case-control) phenotype based on the
+        logistic regression model. Hail supports the Wald test,
+        likelihood ratio test (LRT), and Rao score test. Hail only
+        includes samples for which phenotype and all covariates are
+        defined. For each variant, Hail imputes missing genotypes as
+        the mean of called genotypes.
+
+        Assuming there are sample annotations ``sa.pheno.isCase``,
+        ``sa.cov.age``, ``sa.cov.isFemale``, and ``sa.cov.PC1``, the
+        command:
+
+        >>> vds.logreg('sa.pheno.isCase', covariates='sa.cov.age,sa.cov.isFemale,sa.cov.PC1')
+
+        considers a model of the form
+
+        .. math::
+        
+          \mathrm{Prob}(\mathrm{isCase}) = \mathrm{sigmoid}(\\beta_0 + \\beta_1 \, \mathrm{gt} + \\beta_2 \, \mathrm{age} + \\beta_3 \, \mathrm{isFemale} + \\beta_4 \, \mathrm{PC1} + \\varepsilon), \quad \\varepsilon \sim \mathrm{N}(0, \sigma^2)
```

Comment:
This isn't rendering properly. Not sure what changed since last commit. Is this a read the docs issue?